test/data/jwt_keys/public_key.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
Would prefer this to be generated from the read secret using openssl

Reply:
It will make baas_client.dart more complicated. I think it is no more needed.